src/margo-api-reference/workload-api/application-package-api/application-description.linkml.yaml

```diff
@@ -201,10 +201,152 @@ classes:
   DeploymentProfile:
     description: Represents a deployment configuration for the application.
     rank: 60
+    attributes:
+      id:
+        description: >-
+          An identifier for the deployment profile, given by the application developer, used to uniquely identify this deployment profile in scope of this application description. 
+          The id can be used to display deployment profiles in a user interface and being able to unique distinguish each one of them.
+        rank: 10
+        range: string
+        required: true
+      description:
+        description: >-
+          This human-readable description of a deployment profile allows to provide additional context about the deployment profile. E.g., the application developer can use this to provide information about the deployment profile's purpose,
+          such as the intended use case. Additionally, the application developer can use this to provide further details on resources, peripherals and interfaces required to run the application.
+        rank: 20
+        range: string
+        required: false
+      requiredResources:
```

Comment:
Is this a list or a dictionary, then add `multivalued: true` and probably `inlined: true` and `inlined_as_list: false/true` (default is `true`, but I'd nevertheless add it).